scss/components/_banner.scss

```diff
@@ -52,6 +52,19 @@
         }
     }
 
+    &__heading {
+        font-size: 30px;
+        line-height: 1.33;
+        color: $mine-shaft;
+        letter-spacing: -0.02px;
+        margin-left: $col;
+        
+        @include breakpoint(sm) {
+            font-size: 24px;
+            margin-left: 0;
+        }
+    }
+
     &__body {
         display: flex;
         display: -ms-flexbox;
```

Comment:
I know not in this PR so please feel free to ignore this commend.
Is this needed, I assumed this works the same was as Stylus and other CSS preprocessors? no need for vendor prefixes such as `webkit`/`moz`/`ms` as these will be generated when it gets turned into CSS.

Reply:
I'm not sure, actually. It may be one to check in due course and perhaps raise as a separate PR if it isn't necessary to have them. I have a feeling we use vendor prefixes in a few spots so could tidy them all up as a separate story later on